cvl/cvl_api.go

```diff
@@ -517,8 +515,6 @@ func (c *CVL) ValidateEditConfig(cfgData []cmn.CVLEditConfigData) (cvlErr CVLErr
 				CVL_LOG(WARNING, "\nValidateEditConfig(): OP_CREATE - Err = %v ", err1)
 			}
 
-			c.yp.SetOperation("CREATE")
```

Comment:
Did you add any alternative for this??
CVL exposes the current operation through a fixed 'config false' yang node which can be used in the must expressions. This call was initializing that internal node. Many of the test failures could be due to this change. This automatic operation type xpath is an important feature of cvl and cannot be dropped. It is required for supporting incremental changes through REST/gNMI APIs.

Reply:
Pretty sure this is actually unutilized.  This created a detached libyang lyd_node root separate from anything else.  There was no cross-linking between this node and other roots, it was never imported or merged into any other node.  It seemed like some sort of dangling implementation detail that is no longer in use. 

Do you have an example of where it was used so I can look at those test cases?

Reply:
There will be many test yangs under sonic-mgmt-common/cvl/testdata/schema which are used by cvl gotests. Many of the test cases would be exercising them. But, may be right about it not being utilized. Cvl does not depend on libyang for must expression validations; so this node may not be needed now.
Easy way to chek is to remove only these 3 SetOperation() calls here and run all the tests with libyang1.